Gemfile

```diff
@@ -64,4 +64,5 @@ end
 spree_opts = { github: 'spree/spree', branch: 'main' }
 gem 'spree_core', spree_opts
 gem 'spree_api', spree_opts
+gem 'sidekiq'
```

Comment:
Please don't add sidekiq as a dependency - Spree::Core defines a `Spree::ApplicationJob` as a base for async jobs. It's based on ApplicationJob, which means that if an application already uses a different job queue, they will be able to keep using it.

Reply:
Makes sense. Thanks!